pkg/search/postgres/aggr_func.go

```diff
@@ -0,0 +1,27 @@
+package postgres
+
+import "github.com/stackrox/rox/pkg/postgres/walker"
+
+// AggrFunc is the internal enum representation of the SQL aggregate functions.
+type AggrFunc string
+
+// Defines all the SQL aggregate functions.
+const (
+	Count AggrFunc = "count"
```

Comment:
Define another one? Empty AggrFunc = ""

Reply:
+1 to this, defining an empty value and using that for comparison instead of comparing to `""`

Reply:
good point